addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,136 @@
+
+```php
+Group    iCloud_Group
+
+String   iCloud_Account_Owner             "iCloud Account Owner [%s]"                    (iCloud_Group) {channel="icloud:account:myaccount:owner"}
+Number   iCloud_Account_NumberOfDevices   "iCloud Account NumberOfDevices [%d]"          (iCloud_Group) {channel="icloud:account:myaccount:numberOfDevices"}
+Switch   iCloud_Account_ForceRefresh      "iCloud Account Force Refresh"                 (iCloud_Group) {channel="icloud:account:myaccount:refresh"}
+
+String   iPhone_BatteryStatus             "Battery Status [%s %%]"             <battery> (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:batteryStatus"}
+Number   iPhone_BatteryLevel              "Battery Level [%.0f]"               <battery> (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:batteryLevel"}
+Switch   iPhone_FindMyPhone               "Trigger Find My iPhone"                       (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:findMyPhone"}
+Location iPhone_Location                  "Coordinates"                                  (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:location"}
+Number   iPhone_LocationAccuracy          "Coordinates Accuracy [%.0f m]"                (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:locationAccuracy"}
+DateTime iPhone_LocationLastUpdate        "Last Update [%1$td.%1$tm.%1$tY, %1$tH:%1$tM]" (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:locationLastUpdate"}
+Number   iPhone_DistanceFromHome          "Distance from home [%.0f m]"                  (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:distanceFromHome"}
+String   iPhone_Street                    "Street [%s]"                                  (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:addressStreet"}
+String   iPhone_City                      "City [%s]"                                    (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:addressCity"}
+String   iPhone_Country                   "Country [%s]"                                 (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:addressCountry"}
+String   iPhone_FormattedAddress          "Address [%s]"                                 (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:formattedAddress"}
+String   iPhone_DeviceName                "Device Name [%s]"                             (iCloud_Group) {channel="icloud:device:myaccount:myiPhone8:deviceName"}
+```
+
+# Configuration with Paper UI
+
+First select iCloud Binding in "Paper UI > Configuration > Things" and provide your id, password and refresh interval. Devices are automatically discovered once the thing is configured and online.
+
+1.  Select the "iCloud Binding": ![Select binding](./doc/Config_1.png "Step 1")
+2.  Select the "iCloud Account: ![Select bridge](./doc/Config_2.png "Step 2") 
+3.  Configure your account and the desired refresh rate. ![Configure](./doc/Config_3.png "Step 3") 
+
+# Logging
```

Comment:
Please remove this section. Logging configuration is out of the scope of a Binding README.